_pytest/runner.py

```diff
@@ -494,9 +494,12 @@ def importorskip(modname, minversion=None):
     """
     __tracebackhide__ = True
     compile(modname, '', 'eval') # to catch syntaxerrors
+    should_skip = False
     try:
         __import__(modname)
     except ImportError:
+        should_skip = True
+    if should_skip:
```

Comment:
Please add a note/issue about that, it's a unrelated bug wrt traceback filtering